akka-http-core/src/main/scala/akka/http/impl/util/LogByteStringTools.scala

```diff
@@ -17,14 +17,14 @@ private[akka] object LogByteStringTools {
 
   def logByteStringBidi(name: String, maxBytes: Int = MaxBytesPrinted): BidiFlow[ByteString, ByteString, ByteString, ByteString, NotUsed] =
     BidiFlow.fromFlows(
-      logByteString(s"$name DOWN", maxBytes),
-      logByteString(s"$name UP  ", maxBytes)
+      logByteString(s"$name FromNet", maxBytes),
```

Comment:
things were inversed here; since the generic one just used up / down, but up/down "mean things" for http, so let's use more specific names here

Reply:
I think it doesn't make sense to change it for `logByteStringBidi` because it might then just be wrong (where before it was arbitrary), depending on the direction in which the stage is used.